hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AtttributeResolver.kt

```diff
@@ -13,36 +13,15 @@ import kotlin.reflect.jvm.javaField
 
 typealias Attributes = Map<String, Attribute>
 
-val FUZZY_ATTRIBUTES = listOf(
-  "firstName",
-  "middleNames",
-  "lastName",
-  "prisonName",
-  "mostSeriousOffence",
-  "locationDescription",
-  "dischargeHospitalDescription",
-  "dischargeDetails",
-  "tattoos.comment",
-  "scars.comment",
-  "marks.comment",
-  "aliases.firstName",
-  "aliases.middleNames",
-  "aliases.lastName",
-)
-
-class Attribute(
-  val type: KClass<*>,
-  val openSearchName: String,
-  val isNested: Boolean = false,
-  val isFuzzy: Boolean = false,
-)
-
 @Configuration
 class AttributeResolver {
   @Bean
   fun attributes(): Attributes = getAttributes(Prisoner::class)
 }
 
+/**
+ * Derive all attributes that can be searched for in queries
+ */
```

Comment:
This class is full of cryptic framework style code, so has been littered with comments to try and give people half a chance if they ever need to change it